android/app/src/main/res/layout/activity_speakers.xml

```diff
@@ -142,6 +142,7 @@
                         android:id="@+id/speaker_bio"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
+                        android:paddingBottom="@dimen/padding_medium"
                         android:paddingEnd="@dimen/padding_medium"
                         android:paddingLeft="@dimen/padding_medium"
                         android:paddingRight="@dimen/padding_medium"
```

Comment:
Remove all and just add android:padding then?

Reply:
@iamareebjamal since there is no `paddingTop` I am removing the bottomMargin from the header textview.